### PR TITLE
CD-160

### PR DESCRIPTION
**Pull request by mariemm97**
- Add Cisco meraki Channe Utilization script
- Add script description JSON

### CODE REVIEW

**Review by mohamedkallel — APPROVED** [review]

File: examples/http/cisco_meraki_channel_utilisation.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+ * @returns {Promise} A promise that resolves with an array of channel utilization information
+ */
+function getChannelUtilization(networksInfo) {
+    var promises = networksInfo.map(function(network) {
+        var d = D.q.defer();
+        var config = {
+            url: "/api/v1/networks/" + network.id + "/networkHealth/channelUtilization",
+            protocol: "https",
+            headers: {
+                "Authorization": "Bearer " + D.device.password(),
```
Comment (GeorgiTsatsev) [inline examples/http/cisco_meraki_channel_utilisation.js:113]: This must be the API key

File: examples/http/cisco_meraki_channel_utilisation.js [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+/**
+ * Function to retrieve channel utilization data for each network.
+ * @param {Array} networksInfo Array of network information.
+ * @returns {Promise} A promise that resolves with an array of channel utilization information
+ */
+function getChannelUtilization(networksInfo) {
+    var promises = networksInfo.map(function(network) {
+        var d = D.q.defer();
+        var config = {
+            url: "/api/v1/networks/" + network.id + "/networkHealth/channelUtilization",
```
Comment (GeorgiTsatsev) [inline examples/http/cisco_meraki_channel_utilisation.js:110]: You should be using the timestamp params if you are only taking a record of the last entries per channel

Comment (GeorgiTsatsev) [reply examples/http/cisco_meraki_channel_utilisation.js:110]: Moreover we will most likely need to only fetch the last 10 minutes (also make sure our sample period is the same) and add that as part of the documentation to explain that the data is actually in 10 min intervals.

File: examples/http/cisco_meraki_channel_utilisation.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    "expected_variables": {
+        "table": "[8]"
+    },
+    "protocols": [
+        "HTTPS"
+    ],
+    "tested_on": [
+        {
+            "name": "Cisco Meraki",
+            "version": "wireless-25-13"
```
Comment (GeorgiTsatsev) [inline examples/http/cisco_meraki_channel_utilisation.json:32]: What is this version? I'd say its tested against cisco meraki dashboard api v1

File: examples/http/cisco_meraki_channel_utilisation.js [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+var table = D.createTable(
+    "Channel Utilization",
+    [
+        { label: "Network", valueType: D.valueType.STRING },
+        { label: "Device", valueType: D.valueType.STRING },
+        { label: "Channel", valueType: D.valueType.STRING },
+        { label: "Channel Utilization", unit: "%", valueType: D.valueType.NUMBER },
+        { label: "Wifi Utilization",  unit: "%", valueType: D.valueType.NUMBER },
+        { label: "Non Wifi Utilization",  unit: "%", valueType: D.valueType.NUMBER },
+        { label: "Start Timestamp", valueType: D.valueType.DATETIME },
```
Comment (GeorgiTsatsev) [inline examples/http/cisco_meraki_channel_utilisation.js:44]: the start and end timestamps are a bit strange to have, I would suggest removing the columns

**Review by mohamedkallel — APPROVED** [review]

**Review by GeorgiTsatsev — APPROVED** [review]

**Pull request merged**